pac4j-oauth/src/main/java/org/pac4j/oauth/profile/generic/GenericOAuth20Profile.java

```diff
@@ -0,0 +1,13 @@
+package org.pac4j.oauth.profile.generic;
+
+import org.pac4j.oauth.profile.OAuth20Profile;
+
+/**
+ * This class is the generic OAuth2 profile definition.
+ * 
+ * @author julioa
+ */
+public class GenericOAuth20Profile extends OAuth20Profile {
```

Comment:
I'm in favor of using the `OAuth20Profile` directly (by unmaking it `abstract`).